example/avoidance/avoidance.cpp

```diff
@@ -0,0 +1,175 @@
+/**
+ * @file avoidance.cpp
+ *
+ * @brief Example that demonstrates the usage of Avoidance plugin.
+ * The example registers with the FakeAvoidanceMediator which keeps track
+ * of the current trajectory. Callbacks with new trajectory information from PX4 are
+ * uploaded to the mediator. The mediator then "calculates" a new trajectory
+ * in real time and uploads that trajectory to the vehicle.
+ *
+ * @author Kalyan Sriram <coder.kalyan@gmail.com>
+ * @date 2020-01-29
+ */
+
+#include <chrono>
+#include <mavsdk/mavsdk.h>
+#include <mavsdk/plugins/action/action.h>
+#include <mavsdk/plugins/follow_me/follow_me.h>
```

Comment:
I assume this will need some more work because right now it's just a copy of the follow me plugin :smile:.